org.eclipse.winery.repository.rest/src/main/java/org/eclipse/winery/repository/rest/resources/servicetemplates/plans/PlansResource.java

```diff
@@ -38,17 +30,24 @@
 import org.eclipse.winery.repository.Constants;
 import org.eclipse.winery.repository.backend.RepositoryFactory;
 import org.eclipse.winery.repository.backend.constants.MediaTypes;
+import org.eclipse.winery.repository.configuration.Environment;
 import org.eclipse.winery.repository.rest.RestUtils;
 import org.eclipse.winery.repository.rest.resources._support.collections.EntityCollectionResource;
 import org.eclipse.winery.repository.rest.resources._support.collections.withid.EntityWithIdCollectionResource;
 import org.eclipse.winery.repository.rest.resources.servicetemplates.ServiceTemplateResource;
-
-import com.sun.jersey.core.header.FormDataContentDisposition;
-import com.sun.jersey.multipart.FormDataBodyPart;
-import org.apache.commons.lang3.StringUtils;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import javax.ws.rs.*;
```

Comment:
Unfortunately, you did not follow http://eclipse.github.io/winery/dev/config/IntelliJ%20IDEA/ completely. Please see "3. Configure the code style".

After importing and activating the code style, please reformat this file again to ensure no imports with `*` are there and the correct import order is done.

Reply:
Done.

By the way, there are a lot of start imports throughout the project. Check it with IntelliJ and "Find in Path" using the following Regex `import .+\.\*;`. Shall I fix it?